mvlearn/decomposition/groupica.py

```diff
@@ -224,6 +224,12 @@ def transform(self, Xs, y=None):
         y : None
             Ignored variable.
 
+        indexes: None, or np array
+            This has to be used when less views are used in
+            transform than during the fit.
+            View Xs[i] should correspond to the view indexes[i]
+            in the fit
```

Comment:
I don't like this. but I understand the need. In this case would be accept something like

`Xs = [X[0], None, X[1]]`

so you pass a list and some views can be None?

Reply:
This had been included in the `BaseCCA` implementation, see https://github.com/mvlearn/mvlearn/blob/688548b55ca3b61ea4bcc3b339cee140d03c0adc/mvlearn/embed/base.py#L139

allowing for a single view to be transformed.

Reply:
Ok so @rflperry  you think the current approach is the one to keep ?
@agramfort we could support the None in the list if you prefer !

Reply:
Not really a downside to supporting None that I can see. And adds valuable utility.

Reply:
Ok so let's do that instead. I'll update the API.

Reply:
I would rather have both things supported actually. It is much simpler to specify indexes than having to had None everywhere.

Reply:
@agramfort There is also the behavior in inverse_transform that should be discussed. I think indexes are good because it only returns arrays that are wanted and in this case it is not obvious to use the None trick

Reply:
I am content with `transform` having an `indexes` (perhaps `indices`) parameter which allows either an integer or a list of integers. I can see potential scenarios where allowing `None` views in `Xs` but I don't think it's a necessary implementation.

Reply:
So if @agramfort agrees, I'd like to keep the `indexes` parameter so that `transform` and `inverse_transform` use the same API and delay the support of None to another pull request.
I'll change its name to indices